Makefile

```diff
@@ -517,8 +517,8 @@ LOOP_SERVER := $(shell echo $${LOOP_SERVER-../loop-server})
 # Either path to the browser, or one of nightly, aurora, beta, firefox.
 TEST_BROWSER := $(shell echo $${TEST_BROWSER-nightly})
 
-ifdef TEST_E10S
-E10S_ARGS = --e10s
+ifndef TEST_E10S
+E10S_ARGS = --disable-e10s
```

Comment:
It'd be good to have a comment about why we've disabled e10s, and info (pointing to a bug, perhaps?) about when we plan to re-enable, given that e10s is going to be the default mode for most users after a while...